packages/components/src/Accordion/Accordion.test.tsx

```diff
@@ -129,4 +131,43 @@ describe('Accordion', () => {
     })
     expect(handleKeyUp).toHaveBeenCalled()
   })
+
+  describe('warnings', () => {
+    beforeEach(() => {
+      global.console = {
+        ...globalConsole,
+        warn: jest.fn(),
+      }
+    })
+
+    test('warns if isOpen is provided without toggleOpen prop', () => {
+      global.console = {
+        ...globalConsole,
+        warn: jest.fn(),
+      }
+
+      renderWithTheme(
+        <Accordion isOpen={true} content="My Accordion Content">
+          My Accordion Label
+        </Accordion>
+      )
+
+      // eslint-disable-next-line no-console
+      expect(console.warn).toHaveBeenCalled()
```

Comment:
Reset `global.console`:
```suggestion
      expect(console.warn).toHaveBeenCalled()
      global.console = globalConsole
```

Reply:
Ended up just adding an `afterEach` call to handle this since I added a second warn test.